src/libs/actions/IOU.js

```diff
@@ -344,7 +344,9 @@ function buildOnyxDataForMoneyRequest(
                 ...(isNewChatReport
                     ? {
                           [chatCreatedAction.reportActionID]: {
-                              errors: ErrorUtils.getMicroSecondOnyxError('iou.error.genericCreateFailureMessage'),
+                              errors: _.isEmpty(transaction.receipt)
```

Comment:
As these 3 lines are duplicated several times, can we create a method like `getMicroSecondIouError(receipt)`?

Reply:
Hmmm. Is it worth it? We'll still have the call 3 times (1 line instead of 2) 🤷